readthedocs/rtd_tests/tests/test_sync_versions.py

```diff
@@ -219,6 +219,94 @@ def test_delete_version(self):
             Version.objects.filter(slug='external').exists(),
         )
 
+    def test_update_stable_version_type(self):
+        self.pip.update_stable_version()
+        stable_version = self.pip.get_stable_version()
+        self.assertEqual(stable_version.type, TAG)
+
+        branches_data = [
+            {
+                "identifier": "master",
+                "verbose_name": "master",
+            },
+            {
+                "identifier": "1.0",
+                "verbose_name": "1.0",
+            },
+            {
+                "identifier": "1.1",
+                "verbose_name": "1.1",
+            },
+            {
+                "identifier": "2.0",
+                "verbose_name": "2.0",
+            },
+        ]
+
+        # Deactivate all other versions, so we only have branches for consideration
+        # for the new stable version.
+        self.pip.versions.exclude(slug__in=[LATEST, STABLE]).update(active=False)
+        sync_versions_task(
+            self.pip.pk,
+            branches_data=branches_data,
+            tags_data=[],
+        )
+
+        self.pip.update_stable_version()
+        stable_version = self.pip.get_stable_version()
+        self.assertEqual(stable_version.type, BRANCH)
+        self.assertEqual(stable_version.identifier, "2.0")
+        self.assertEqual(stable_version.verbose_name, "stable")
+
+        original_stable = self.pip.get_original_stable_version()
```

Comment:
I doubted what was the meaning of "original stable", so I went to the docstring: https://github.com/readthedocs/readthedocs.org/blob/bbb200ba2e42d032843e3f8da170c61b76af514e/readthedocs/projects/models.py#L1167-L1172

but it's not super useful. Do you know what this mean? What is the difference between the "original stable" (`get_original_stable_version`) and the "stable" (`get_stable_version`)?

Reply:
When stable is machine created, it's basically an alias for the latest stable version (like 2.2), sometimes we need to point to that version, that's the "original" one.

Reply:
oh, I see. Would you mind adding that comment in the docstring of that method? 🙏🏼